pkg/services/authn/clients/anonymous.go

```diff
@@ -23,6 +25,7 @@ type Anonymous struct {
 	cfg        *setting.Cfg
 	log        log.Logger
 	orgService org.Service
+	kvstore    kvstore.KVStore
```

Comment:
i do not see where this is being used, is this necessary.

I guess it to satisfy an interface? if so; should we add it as `_kvStore`

Reply:
Relevant [conversation](https://raintank-corp.slack.com/archives/C043E603N8M/p1675673277959959?thread_ts=1675673124.096639&cid=C043E603N8M). This will be used in one of the upcoming PRs to generate stats on anonymous sessions.